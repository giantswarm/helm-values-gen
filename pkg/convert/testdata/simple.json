{
  "$schema": "https://json-schema.org/draft/2020-12/schema",
  "properties": {
    "age": {
      "type": "integer",
      "default": 21
    },
    "lastName": {
      "type": "string",
      "default": "Doe"
    },
    "height": {
      "type": "number",
      "default": 1.78
    },
    "isAdult": {
      "type": "boolean",
      "default": true
    }
  },
  "type": "object"
}
