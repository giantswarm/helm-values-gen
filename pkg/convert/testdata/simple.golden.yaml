age: 21
height: 1.78
isAdult: true
lastName: Doe
